src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Process/Module.ts

```diff
@@ -13,6 +13,7 @@ export var register = (angular) => {
         .provider("adhProcess", AdhProcess.Provider)
         .directive("adhWorkflowSwitch", ["adhConfig", "adhHttp", "adhPermissions", "$window", AdhProcess.workflowSwitchDirective])
         .directive("adhProcessView", ["adhTopLevelState", "adhProcess", "$compile", AdhProcess.processViewDirective])
+        .directive("adhButtonSlot", ["adhTopLevelState", "adhProcess", "$compile", AdhProcess.processButtonSlot])
```

Comment:
this does not apply here: `processButtonSlot` was always used as a property of `adhProcessProvider`, so the "process" is redundant. But `adhButtonSlot` is used on its own.